FourthTask/sorting/main/main.go

```diff
@@ -0,0 +1,34 @@
+package main
```

Comment:
You don't have to create a folder `main` to hold a package `main`.
Feel free to create a separate folder for a task, create a `main` package there and subpackages with the names you want. 
You can ping me in slack if you need some help on it.